app/dashboard/views.py

```diff
@@ -1237,15 +1237,15 @@ def profile(request, handle):
 
             paginator = Paginator(activities, 10)
 
-            obj = {}
-            obj['id'] = tab
-            obj['name'] = name
-            obj['activities'] = paginator.get_page(1)
-            obj['count'] = activities_count
+            obj = {'id': tab,
```

Comment:
nice cleanup! 👍

Reply:
Thanks @danlipert I'm also having an error with this branch not sure why only in this one since looks like is not related to the code in the branch.